nodejs/awsx/apigateway/README.md

```diff
@@ -0,0 +1,139 @@
+# Pulumi API Gateway Components
+
+Pulumi's API for simplifying working with [API Gateway](https://aws.amazon.com/api-gateway/). The API currently provides ways to define a route that accepts and forwards traffic to a specified destination.
+
```

Comment:
maybe "define routes that accept".  "route"-singular here makes it sound like you can only create one route.

Reply:
It might make sense to explain what a route (or routing) is.  i.e. that it's goign to be an accesssible http address (or URI) that supports some http method (i.e. GET/POST/etc.)  to access it, and which will respond in some specialized way when the http address is hit.

Reply:
nit: maybe "supports a specified HTTP method ('GET', 'POST', etc.)".